### ISSUE TITLE
Bug: IndexError: list index out of range

### ISSUE THREAD

**Issue by L0Lock** [cdgriffith/FastFlix#60]
Not sure exactly what causes the issue, but fastflix can't work with some of my files. I haven't thoroughly tested, but it looks like it occurs with files containing multiple audio/subtitles steams. Maybe...

When using those files, the command list tab remains blank, and using the convert button doesn't do anything besides setting an output file.

This is the log output:

```
DEBUG - 2020-09-14 00:19:47,955 - running probe command: "C:\AppInstl\ffmpeg\bin\ffprobe.EXE" -v quiet -print_format json -show_format -show_streams "S:/temp/test.mkv"
DEBUG - 2020-09-14 00:19:48,969 - running command: "C:\AppInstl\ffmpeg\bin\ffprobe.EXE" -v quiet -print_format json -show_format -show_streams "S:/temp/test.mkv"
DEBUG - 2020-09-14 00:19:49,108 - {'video': [{'index': 3, 'codec_name': 'h264', 'codec_long_name': 'H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10', 'profile': 'High', 'codec_type': 'video', 'codec_time_base': '1001/48000', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'width': 1920, 'height': 800, 'coded_width': 1920, 'coded_height': 800, 'closed_captions': 0, 'has_b_frames': 2, 'sample_aspect_ratio': '1:1', 'display_aspect_ratio': '12:5', 'pix_fmt': 'yuv420p', 'level': 41, 'color_range': 'tv', 'color_space': 'bt709', 'chroma_location': 'left', 'field_order': 'progressive', 'refs': 1, 'is_avc': 'true', 'nal_length_size': '4', 'r_frame_rate': '24000/1001', 'avg_frame_rate': '24000/1001', 'time_base': '1/1000', 'start_pts': 0, 'start_time': '0.000000', 'bits_per_raw_sample': '8', 'disposition': {'default': 1, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}, 'tags': {'language': 'eng', 'BPS-eng': '8559522', 'DURATION-eng': '01:58:58.131000000', 'NUMBER_OF_FRAMES-eng': '171144', 'NUMBER_OF_BYTES-eng': '7637373896', '_STATISTICS_WRITING_APP-eng': "mkvmerge v41.0.0 ('Smarra') 64-bit", '_STATISTICS_WRITING_DATE_UTC-eng': '2020-03-12 07:43:06', '_STATISTICS_TAGS-eng': 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES'}}], 'audio': [{'index': 0, 'codec_name': 'ac3', 'codec_long_name': 'ATSC A/52A (AC-3)', 'codec_type': 'audio', 'codec_time_base': '1/48000', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'sample_fmt': 'fltp', 'sample_rate': '48000', 'channels': 6, 'channel_layout': '5.1(side)', 'bits_per_sample': 0, 'dmix_mode': '-1', 'ltrt_cmixlev': '-1.000000', 'ltrt_surmixlev': '-1.000000', 'loro_cmixlev': '-1.000000', 'loro_surmixlev': '-1.000000', 'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/1000', 'start_pts': 0, 'start_time': '0.000000', 'bit_rate': '448000', 'disposition': {'default': 1, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}, 'tags': {'language': 'fre', 'BPS-eng': '448000', 'DURATION-eng': '01:58:58.144000000', 'NUMBER_OF_FRAMES-eng': '223067', 'NUMBER_OF_BYTES-eng': '399736064', '_STATISTICS_WRITING_APP-eng': "mkvmerge v41.0.0 ('Smarra') 64-bit", '_STATISTICS_WRITING_DATE_UTC-eng': '2020-03-12 07:43:06', '_STATISTICS_TAGS-eng': 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES'}}, {'index': 4, 'codec_name': 'ac3', 'codec_long_name': 'ATSC A/52A (AC-3)', 'codec_type': 'audio', 'codec_time_base': '1/48000', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'sample_fmt': 'fltp', 'sample_rate': '48000', 'channels': 6, 'channel_layout': '5.1(side)', 'bits_per_sample': 0, 'dmix_mode': '-1', 'ltrt_cmixlev': '-1.000000', 'ltrt_surmixlev': '-1.000000', 'loro_cmixlev': '-1.000000', 'loro_surmixlev': '-1.000000', 'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/1000', 'start_pts': 0, 'start_time': '0.000000', 'bit_rate': '640000', 'disposition': {'default': 0, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}, 'tags': {'language': 'eng', 'BPS-eng': '640000', 'DURATION-eng': '01:58:58.144000000', 'NUMBER_OF_FRAMES-eng': '223067', 'NUMBER_OF_BYTES-eng': '571051520', '_STATISTICS_WRITING_APP-eng': "mkvmerge v41.0.0 ('Smarra') 64-bit", '_STATISTICS_WRITING_DATE_UTC-eng': '2020-03-12 07:43:06', '_STATISTICS_TAGS-eng': 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES'}}], 'subtitle': [{'index': 1, 'codec_name': 'subrip', 'codec_long_name': 'SubRip subtitle', 'codec_type': 'subtitle', 'codec_time_base': '0/1', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/1000', 'start_pts': 0, 'start_time': '0.000000', 'duration_ts': 7138144, 'duration': '7138.144000', 'disposition': {'default': 1, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 1, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}, 'tags': {'language': 'fre', 'title': 'Forced', 'BPS-eng': '1', 'DURATION-eng': '01:57:50.855000000', 'NUMBER_OF_FRAMES-eng': '36', 'NUMBER_OF_BYTES-eng': '1618', '_STATISTICS_WRITING_APP-eng': "mkvmerge v41.0.0 ('Smarra') 64-bit", '_STATISTICS_WRITING_DATE_UTC-eng': '2020-03-12 07:43:06', '_STATISTICS_TAGS-eng': 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES'}}, {'index': 2, 'codec_name': 'subrip', 'codec_long_name': 'SubRip subtitle', 'codec_type': 'subtitle', 'codec_time_base': '0/1', 'codec_tag_string': '[0][0][0][0]', 'codec_tag': '0x0000', 'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/1000', 'start_pts': 0, 'start_time': '0.000000', 'duration_ts': 7138144, 'duration': '7138.144000', 'disposition': {'default': 0, 'dub': 0, 'original': 0, 'comment': 0, 'lyrics': 0, 'karaoke': 0, 'forced': 0, 'hearing_impaired': 0, 'visual_impaired': 0, 'clean_effects': 0, 'attached_pic': 0, 'timed_thumbnails': 0}, 'tags': {'language': 'fre', 'title': 'French', 'BPS-eng': '34', 'DURATION-eng': '01:57:50.855000000', 'NUMBER_OF_FRAMES-eng': '944', 'NUMBER_OF_BYTES-eng': '30608', '_STATISTICS_WRITING_APP-eng': "mkvmerge v41.0.0 ('Smarra') 64-bit", '_STATISTICS_WRITING_DATE_UTC-eng': '2020-03-12 07:43:06', '_STATISTICS_TAGS-eng': 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES'}}], 'attachment': [], 'data': []}
DEBUG - 2020-09-14 00:19:49,111 - {'filename': 'S:/temp/test.mkv', 'nb_streams': 5, 'nb_programs': 0, 'format_name': 'matroska,webm', 'format_long_name': 'Matroska / WebM', 'start_time': '0.000000', 'duration': '7138.144000', 'size': '8610170356', 'bit_rate': '9649758', 'probe_score': 100, 'tags': {'encoder': 'libebml v1.3.9 + libmatroska v1.5.2', 'creation_time': '2020-03-12T07:43:06.000000Z'}}
INFO - 2020-09-14 00:19:49,113 - height has -0px off aspect
DEBUG - 2020-09-14 00:19:49,114 - 1 video tracks found
DEBUG - 2020-09-14 00:19:49,114 - 2 audio tracks found
DEBUG - 2020-09-14 00:19:49,115 - 2 subtitle tracks found
ERROR - 2020-09-14 00:19:49,209 - Exception in 'update_video_info': list index out of range - args: (,)
ERROR - 2020-09-14 00:19:49,210 - Exception in 'dropEvent': list index out of range - args: (, )
```

I also have this in the console:
```batch
2020-09-14 00:19:49,210 - fastflix  ERROR    Exception in 'dropEvent': list index out of range - args: (<fastflix.widgets.main.Main(0x23bb2516f10) at 0x0000023BB2F45D88>, <PySide2.QtGui.QDropEvent object at 0x0000023BB5344F88>)
Traceback (most recent call last):
  File "site-packages\reusables\wrappers.py", line 270, in wrapper
  File "site-packages\reusables\wrappers.py", line 262, in wrapper
  File "fastflix\widgets\main.py", line 864, in dropEvent
  File "site-packages\reusables\wrappers.py", line 270, in wrapper
  File "site-packages\reusables\wrappers.py", line 262, in wrapper
  File "fastflix\widgets\main.py", line 649, in update_video_info
  File "fastflix\widgets\video_options.py", line 63, in new_source
  File "fastflix\plugins\av1\settings_panel.py", line 147, in new_source
  File "site-packages\box\box_list.py", line 69, in __getitem__
IndexError: list index out of range
```

**Comment by cdgriffith** [comment]
Thank you for including the log output, made finding the issue really simple! 

It turns out was an issue with the video track not being the first track, which isn't common to my knowledge so I didn't have a test case for that. Fix will be in 2.6.0.

**Comment by cdgriffith** [comment]
Should be fixed in 2.6.0 https://github.com/cdgriffith/FastFlix/releases/tag/2.6.0 let me know if you have any more issues, thanks!

**Comment by L0Lock** [comment]
Looks like it's fixed so far. Many thanks sir!
